src/problems.jl

```diff
@@ -107,121 +87,34 @@ function conic_form!(p::Problem, unique_conic_forms::UniqueConicForms=UniqueConi
     return objective, objective_var.id_hash
 end
 
-function conic_problem(p::Problem)
```

Comment:
I may have missed it, but does this function not exist anymore? It's still referenced in the updated docs, but that appears to be it?

Reply:
Cool, that all seems reasonable. I just wanted to clarify that `load_MOI_model!` was the closest thing to what this previously did and that we want to remove that reference from the docs. Thanks for fixing that up already 👍